docusaurus/docs/core-concepts/clients/tunnelers/04-macos.md

```diff
@@ -1,3 +1,41 @@
 # MacOS
 
-The MacOS tunneler is currently in preview. Find the app in the [Mac App Store](https://apps.apple.com/us/app/ziti-tunnel/id1460484572?mt=12)
+## Installation
+
+Install the **Ziti Desktop Edge** from the [Mac App Store](https://apps.apple.com/us/app/ziti-tunnel/id1460484572?mt=12)
+
+
+:::info Note
+
+If you already have MAC Ziti Desktop already installed and you are updating it, please make sure to turn off your MAC Ziti Desktop Edge before updating it

```

Comment:
"MAC Ziti Desktop" should be "Ziti Desktop Edge for Mac" and "updating it" is redundant. Let's go with :

> If you already have the __Ziti Desktop Edge for Mac__ installed, it is a best practice to disconnect from the overlay network before updating. Do that by clicking _"Turn Ziti Off"_ from the UI or choose _"Disconnect"_ from the icon in the notification area

Reply:
other than the top one - there are a few other places that should be "Ziti Desktop Edge for Mac" in this doc. find/replace those please

Reply:
ok